CONTRIBUTING.md

```diff
@@ -169,7 +169,9 @@ We prefer concise names, especially for local variables. Avoid adding a suffix
 for a variable that describes its type (provided that its type is hard to
 confuse with other types -- for example, we do still use `_id` suffixes because
 we usually use numeric IDs for database entities). The precision/conciseness
-trade-off for variable names also depends on the scope of the binding.
+trade-off for variable names also depends on the scope of the binding. In
+general we prefer to expand acronyms (e.g. using `keyusage` instead of `ku`,
```

Comment:
`keyusage` is a bad example because we should snake_case it: `key_usage`. This feels a little tacked on to this paragraph... maybe it should go up front, after the first sentence -- especially since it is kind of at odds with it. Might also worth adding a note about how well-known the acronym/abbreviation is. For example, I would not hesitate to call a variable `html` or `url` but I think `ku` or `eku` are indeed relatively obscure.

Reply:
Fixed `keyusage` -> `key_usage`, moved to the front, and added a note about very common counterexamples.